src/bits.c

```diff
@@ -1147,22 +1147,32 @@ bit_read_CRC(Bit_Chain * dat)
 /** Read and check CRC-number.
  */
 int
-bit_check_CRC(Bit_Chain * dat, long unsigned int start_address,
-              uint16_t seed)
+bit_check_CRC(Bit_Chain * dat, long unsigned int start_address, 
+              long unsigned int size, uint16_t seed)
 {
   unsigned int calculated;
   unsigned int read;
   unsigned char res[2];
+  long unsigned int old_byte;
+  long unsigned int old_bit;
```

Comment:
unneeded since it always must be 0. We always align dat for CRC calculations

Reply:
There is a `start_address` parametr. It always align (directly bytes access by `dat->chain + start_address`). Input `dat` may not be not alligned in general case?